### PR TITLE
allow to visit Single Circle

### PR DESCRIPTION
**Pull request by ArtificialOwl**
Will be useful when searching for users as entity

### CODE REVIEW

**Comment by ArtificialOwl** [comment]
@nextcloud/server-backend please review !

File: lib/Model/Probes/CircleProbe.php [hunk 1]
```diff
@@ -126,6 +130,25 @@ public function includeBackendCircles(bool $include = true): self {
 	}
 
 
+	/**
+	 * @param bool $visit
+	 *
+	 * @return $this
+	 */
+	public function visitSingleCircles(bool $visit = true): self {
```
Comment (skjnldsv) [inline lib/Model/Probes/CircleProbe.php:138]: A bit of documentation on what it does? :)

Comment (ArtificialOwl) [reply lib/Model/Probes/CircleProbe.php:138]: So, the provisioning API for Circles (for NC23+) should be able to returns all entities like Circles and Circles generated from Nextcloud Groups. However, with the current status of the code, the Circles App have no way to browse the list of singleIds for Nextcloud Users (as Entities).

This methods visitSingleCircles() allow to add a parameter to the Probe to that the request will also returns singleIds for Nextcloud Users (as a visitor as the current viewer/initiator does not belongs to the Single Circle)

Comment (skjnldsv) [reply lib/Model/Probes/CircleProbe.php:158]: No, I meant code documentation :wink: 
Your functions on this are missing docs headers :see_no_evil:

Comment (ArtificialOwl) [reply lib/Model/Probes/CircleProbe.php:158]: i will write this docs in the next big PR about public interface

Comment (skjnldsv) [reply lib/Model/Probes/CircleProbe.php:158]: Well, all functions should have a documentation header. It's not about being public or not :wink:

Comment (mejo-) [reply lib/Model/Probes/CircleProbe.php:174]: @skjnldsv I agree that a short descriptive header is helpful to understand the meaning functions. Though this class is probably a bad example to start. Still, I added documentation headers for the functions: https://github.com/nextcloud/circles/pull/783/commits/369cfe5ed757acf463357b6ed8ad704863eed84c

**Review by mejo- — APPROVED** [review]

**Comment by ArtificialOwl** [comment]
no need to backport

**Pull request merged**